modules/core/classes/GalleryTemplateAdapter.class

```diff
@@ -870,9 +870,10 @@ class GalleryTemplateAdapter {
 	$buf = '';
 	if ($debug == 'buffered' || $profiling) {
 	    /* don't worry about localizing this -- it's only debug output */
-	    $buf .= '<div id="gpDebug"> <h3> Debug Output </h3>';
+	    $buf .= '<div id="gpDebug"> <h3> Debug Output </h3><pre>';
 	    if ($debug == 'buffered') {
 		$buf .= $gallery->getDebugBuffer();
+	    	$buf .= '</pre>';
```

Comment:
Hmm, it seems like now the `<pre>` tag might not be closed if the debug isn't buffered. Can you just move this line out of the `if` block?